api/proto/node/node.proto

```diff
@@ -54,6 +54,14 @@ message RetrieveChunksRequest {
 message RetrieveChunksReply {
 	// All chunks the Node is storing for the requested blob per RetrieveChunksRequest.
 	repeated bytes chunks = 1;
+	// Describes how the chunks above are encoded.
+	// If not specified, it’s treated as GOB.
```

Comment:
if unspecified, wouldn't it be the first definition in enum, which is unknown

Reply:
We need to default to GOB in order to be backward compatible

Reply:
Actually it's fine to default o UNKNOWN -- we just need to make sure the GOB is set when we return GOB encoded chunks